data/omk-coder.json

```
@@ -1,11 +1,26 @@
 {
   "name": "Omkar Ramishte",
   "type": "personal",
-  "bio": "Currently Cs Grad, beginning with open source journey, learner, shares about open source related threads and  Blogs on Twitter. Make sure to check out. Happy to connect❤️ ",
+  "bio": " CS UnderGrad | Frontend Developer | shares about open source related threads and  Blogs on Twitter| Happy to connect❤️ ",
+  
+  "tags": [
+    "C++",
+    "ReactJS",
+    "Javascript",
+    "tailwindCSS",
+    "HTML/CSS",
+    "Open Source" 
+  ],
+  "socials":[
+    {"icons":"FaGithub", "url":"https://github.com/omk-coder"},
+      {"icons":"FaLinkedin", "url":"https://www.linkedin.com/in/omkar-ramishte-9ba784207/"},
+      {"icons":"FaTwitter", "url":"https://twitter.com/ORamishte"},
+      {"icons":"SiHashnode", "url":"https://hashnode.com/@Omkar009"}
```

Should be `icon` here as well.
```suggestion
    {"icon":"FaGithub", "url":"https://github.com/omk-coder"},
      {"icon":"FaLinkedin", "url":"https://www.linkedin.com/in/omkar-ramishte-9ba784207/"},
      {"icon":"FaTwitter", "url":"https://twitter.com/ORamishte"},
      {"icon":"SiHashnode", "url":"https://hashnode.com/@Omkar009"}
```